src/data/real/sign.lean

```diff
@@ -0,0 +1,61 @@
+/-
+Copyright (c) 2021 Kexing Ying. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kexing Ying
+-/
+import data.real.basic
+
+/-!
+# Real sign function
+
+This file introduces and contains some results about `real.sign` which maps negative
+real numbers to -1 and nonnegative numbers to 1.
+
+## Main definitions
+
+ * `real.sign r` is -1 if `r < 0` and 1 otherwise
+
+## Tags
+
+sign function
+-/
+
+namespace real
+
+/-- The sign function that maps negative real numbers to -1 and nonnegative numbers to 1. -/
+noncomputable
+def sign (r : ℝ) : ℝ := if r < 0 then -1 else 1
```

Comment:
Thanks! Would you mind adding simp-lemmas for `sign_one` and `sign_zero`?

Reply:
Done!